docs/guide/content_rendering/render_content/render_content.md

```diff
@@ -0,0 +1,95 @@
+# Render content
+
+Content is rendered automatically using default, basic templates.
+To render content with a custom template, you need to create a template file
+and tell the system, in configuration, when to use this template.
+
+You do it by using the [content view configuration](../templates/template_configuration.md).
```

Comment:
```suggestion
You can do it by using the [content view configuration](../templates/template_configuration.md).
```
or 

```suggestion
This can be done by using the [content view configuration](../templates/template_configuration.md).
```